DearDiarySwift/ContentView.swift

```diff
@@ -0,0 +1,148 @@
+                        .cornerRadius(30)
+                    }
+                }
+                .padding(/*@START_MENU_TOKEN@*/.all, 30.0/*@END_MENU_TOKEN@*/)
+                .background(/*@START_MENU_TOKEN@*//*@PLACEHOLDER=View@*/Color.white/*@END_MENU_TOKEN@*/)
+                .cornerRadius(20)
+            }
+            
+        }
+        //second screen
+        
+        ZStack {
+//                        Image("3175")
+            HStack{
+                NavigationView{
+                    VStack {
+                        Section(header: Text("Your are here: Home")
+                                    .padding(.vertical, 25.0)){
+                            TextField("Submit New",text: $text).padding().frame(width: 300.0, height: 50.0).border(/*@START_MENU_TOKEN@*/Color.black/*@END_MENU_TOKEN@*/, width: /*@START_MENU_TOKEN@*/1/*@END_MENU_TOKEN@*/).cornerRadius(10)
+                            
+                            //                            TextField("Enter Description",text: $text).padding(.bottom, 80.0).frame(width: 300.0, height: 130.0).border(/*@START_MENU_TOKEN@*/Color.black/*@END_MENU_TOKEN@*/, width: /*@START_MENU_TOKEN@*/1/*@END_MENU_TOKEN@*/).cornerRadius(8)
+                            
+                            
+                            Button(action: {
+                                tryToAddList()
+                            }, label: {
+                                Text("Submit").foregroundColor(Color.black).frame(width: 200, height: 50, alignment: /*@START_MENU_TOKEN@*/.center/*@END_MENU_TOKEN@*/)
+                                    .background(/*@START_MENU_TOKEN@*//*@PLACEHOLDER=View@*/Color.blue/*@END_MENU_TOKEN@*/)
+                                    .cornerRadius(25)
+                            })
+                            .padding(.leading, 35.0)
+                            
+                        }
+                        
+                        List {
+                            ForEach(viewModel.notes){
+                                notes in NoteRows(title: notes.title)
+                            }
+                            
+                            
+                        }
+                        .navigationTitle("Home")
+                    }
+                }
+                
+                
+            }
+        }
+        
+    }
+    func tryToAddList() {
+        guard !text.trimmingCharacters(in: .whitespaces).isEmpty else {
+            return
+            
+        }
+        let newNotes = Notes(title: text)
+        viewModel.notes.append(newNotes)
+        text=""
+        
+    }
+    
+}
+
+struct Notes: Identifiable{
+    var id = UUID()
+    let title:String
+}
+class NotesViewModel: ObservableObject{
+    @Published var notes: [Notes]=[
+        Notes(title: "Leneovo"),
+        Notes(title: "Leneovo"),
+        Notes(title: "Leneovo"),
+        Notes(title: "Leneovo"),
+    ]
+    
+}
+
+struct NoteRows: View  {
```

Comment:
- [ ] When you are having custom views like this, have them in separate files, so it's easy to maintain

Reply:
It's easy to navigate & maintain 100 files instead of having a single file with 1000 lines.